list-tags should work in a non git directory

Running...
`yor list-tags` 
...gives the error
`<datetime> [ERROR] Failed to initialize git service for path .. Please ensure the provided root directory is initialized via the git init command.`

After speaking with @schosterbarak it seems the available tags list is not dependent the git repo or local path.  As such, this command should work outside of a git repo.  Or at least show the defaults without any local saved state if that were to be the case.

Happy to work on this issue as well BTW.

--- a/main.go
+++ b/main.go
@@ -94,7 +94,7 @@ func listTags(options *cli.ListTagsOptions) error {
 		if tagGroup == nil {
 			return fmt.Errorf("tag group %v is not supported", group)
 		}
-		tagGroup.InitTagGroup(".", nil)
+		tagGroup.InitTagGroup("", nil)
 		tagsByGroup[group] = tagGroup.GetTags()
 	}
 	reports.ReportServiceInst.PrintTagGroupTags(tagsByGroup)

